app/Http/Controllers/StatusController.php

```diff
@@ -418,4 +419,32 @@ public static function getFutureCheckins(): Paginator {
                          $query->where('departure', '>=', date('Y-m-d H:i:s', strtotime("+20min")));
                      })->simplePaginate(15);
     }
+
+    public static function createStatus(
+        User $user,
+        bool $business,
+        $visibility,
+        string $body = null,
+        int $eventId = null,
+        string $type = "hafas"
+    ): Status {
+        $event = null;
+        if ($eventId !== null) {
+            $event = Event::find($eventId);
+            //FixMe: Is the "new Carbon" really needed here?
+            if (!Carbon::now()->isBetween(new Carbon($event?->begin), new Carbon($event?->end))) {
```

Comment:
new Carbon should not be needed, as begin and end should be an carbon Object already